site/static/compare/script.js

```diff
@@ -6,6 +6,29 @@ function findQueryParam(name) {
     }
 }
 
+function createDefaultFilter() {
+    return {
+        name: null,
+        showNonRelevant: false,
```

Comment:
nit: I think this should be `nonRelevant` as all of the others also have to do with what is "shown" but do not contain the "show" verb.